### DISCUSSION TITLE
email rate limit exceeded

### DISCUSSION THREAD

**Discussion by ilkovulchev** [supabase/supabase#31255] [Questions] [answered]
Hello,

I am testing my password reset functionality and all of a sudden I got a 429 (Too many requests) error with this message: "email rate limit exceeded" in my last reset test. Why is that? Where can I see the limit and also, since I am using a custom SMTP server, shouldn't the limit be from the provider I am using? I am using Brevo, as my SMTP server provider and I have more than enough emails left, to test, as you can see
![image](https://github.com/user-attachments/assets/ed11560b-a010-4fde-bbb2-e1f633441f58)

Any help would be appreciated. Thanks :)

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/06cc1926-ca68-438d-bdd9-2dd856573784)

**Comment by ilkovulchev** [comment]
hello,

Thank you. I just found it out myself. To be honest I saw it before I wrote my question, but the limit was 2 per four and I definitely sent more than 2 in the past hour, before I wrote this. So it seemed strange to me, but when I set to 20 it worked. Is there something I am missing here? I've been testing for the last couple of hours and for sure had sent more than 2 emails per hour. Take a look at my Brevo logs as well
![image](https://github.com/user-attachments/assets/cd185e6d-fbb2-40ef-a6f6-d3787a39d834)

Sorry for being annoying but it's my first time using Supabase and I want to understand why this happened :)

Thank you

**Reply by GaryAustin1** [reply]
I don't know why it would not error at the set amount or maybe 1 over it. 
You could generate an issue in supabase/auth github and ask with your info.  But unless it is reproducible may not get far.
